source/include/cephadm.rst

```diff
@@ -34,14 +34,17 @@ cephadm based playbooks utilising stackhpc.cephadm Ansible Galaxy collection.
 Running Ceph commands
 =====================
 
-Ceph commands can be run via ``cephadm shell`` utility container:
+Ceph commands are usually run inside a ``cephadm shell`` utility container:
 
 .. code-block:: console
 
    ceph# cephadm shell
 
-This command will be only successful on ``mons`` group members (the admin key
-is copied only to those nodes).
+Operating a cluster requires a keyring with an admin access to be available for Ceph
+commands. This is easiest on the nodes that are marked with
```

Comment:
easiest? I think it's not possible on other nodes ;-)

Reply:
you could always have a copy of the keyring placed somewhere else manually

Reply:
Yes, but that explanation is not there for the users - which is the audience of Admin Guide - let's make it explicit that cephadm will copy the admin keyring only to the nodes with _admin label.